pkg/defaults/provider.go

```diff
@@ -17,20 +17,16 @@ import (
 // Base is the base directory inside which all the other directories are
 // created.
 func NewProvider(base string) *Provider {
-	obj := &Provider{Base: base}
-	obj.Init()
-	return obj
+	return &Provider{Base: base, Home: "/var/lib/embedded-cluster"}
```

Comment:
I am very confused by what you are doing in this file

Reply:
I suppose it's OK

Reply:
Yeah, when running this from within a pod we may want to materialise the files into a different location. I don't like this anymore, let me revisit.

Reply:
OK, this has been removed from the PR as now we require `root` and materialise directly to `/var/lib/embedded-cluster`.